grpc/src/compiler/schema_interface.h

```diff
@@ -111,7 +112,8 @@ struct File : public CommentHolder {
   virtual int service_count() const = 0;
   virtual std::unique_ptr<const Service> service(int i) const = 0;
 
-  virtual std::unique_ptr<Printer> CreatePrinter(grpc::string *str) const = 0;
+  virtual std::unique_ptr<Printer> CreatePrinter(
+      grpc::string *str, const char indentation_type = ' ') const = 0;
 };
```

Comment:
I allowed anyone who is using the Printer to actually indicate what type of indentation type they want.